pkg/check/node_hw_version_test.go

```diff
@@ -51,15 +69,14 @@ vsphere_node_hw_version_total{hw_version="vmx-15"} 1
 			defer cleanup()
 
 			// Set HW version of the first VM. Leave the other VMs with the default version (vmx-13).
-			node := kubeClient.nodes[0]
-			err = customizeVM(ctx, node, &types.VirtualMachineConfigSpec{
-				ExtraConfig: []types.BaseOptionValue{
-					&types.OptionValue{
-						Key: "SET.config.version", Value: test.hwVersion,
-					},
-				}})
-			if err != nil {
-				t.Fatalf("Failed to customize node: %s", err)
+			if len(test.hwVersions) > 0 {
+				for i := range test.hwVersions {
```

Comment:
nit: maybe you can just remove the if clause above

Reply:
that sounds fine, but this is PR is a cherry-pick of master branch and hence I am bit hesitant to modify it here.